numba/core/bytecode.py

```diff
@@ -154,6 +174,16 @@ def _unpack_opargs(code):
             for j in range(ARG_LEN):
                 arg |= code[i + j] << (8 * j)
             i += ARG_LEN
+            if PYVERSION in ((3, 12), ):
+                # Python 3.12 introduced cache slots. We need to account for
+                # cache slots when we determine the offset of the next opcode.
+                # The number of cache slots is specific to each opcode and can
+                # be looked up in the _inline_cache_entries dictionary.
+                i += _inline_cache_entries[op] * 2
```

Comment:
Where does `2` come from? Should it be a named constant?

Reply:
`2` is for byte length of an instruction/opcode/bytecode -- can make it a constant, yes

Reply:
fixed in 9353b9a20d040d737381bdb42f22a40a9c9e46e0

Reply:
and in 101fcfb57f5b005932d7e04bd4eed33a728f0058